site/migration/01-migration.md

```diff
@@ -0,0 +1,278 @@
+## Migration from kpt `v0.39.0` to kpt `v1.0`
+
+- [Before you begin](#Before-you-begin)
```

Comment:
In the GitHub preview, these don't show up as links. Not sure if that's a real issue or not.

Reply:
They do for me. Try this out https://github.com/GoogleContainerTools/kpt/blob/51e2495577f27c272210dcd8a17f3090797f2f91/site/installation/migration/01-migration.md#Before-you-begin